kolibri_zim_plugin/assets/src/views/DOMTreeRenderer.vue

```diff
@@ -141,20 +141,10 @@
 
 <style lang="scss" scoped>
 
-  @import '~kolibri-design-system/lib/styles/definitions';
-
   .content {
-    display: block;
-    width: 100%;
-    height: 100%;
-    overflow: auto;
     // Set the transform property so fixed-position children are relative to
     // this element.
     transform: translate(0);
   }
 
-  .load-content {
-    display: none;
-  }
```

Comment:
This is undoing a change done in this very PR. So it deserves squashing with the previous commit.